src/wp-includes/class-wp-query.php

```diff
@@ -5064,7 +5064,7 @@ static function ( &$value ) use ( $wpdb, $placeholder ) {
 
 		$last_changed = wp_cache_get_last_changed( 'posts' );
 		if ( ! empty( $this->tax_query->queries ) ) {
-			$last_changed .= wp_cache_get_last_changed( 'terms' );
+			$last_changed = wp_cache_get_last_changed( 'terms' ) . '+' . $last_changed;
```

Comment:
Why doesn't this append anymore?

Reply:
>I propose this new format, where the primary cache group is always last:

I was proposing to have the primary group's timestamp to always be the last, so we know which one to parse, maybe maybe it should always be the first?

Reply:
I don't have any inclination one way or the other.

Reply:
Either works, but to me having the primary group's timestamp first feels more natural.

Reply:
Will update!